MonkeyChat/ConversationListViewController.swift

```diff
@@ -786,6 +710,167 @@ extension ConversationsListViewController {
   }
 }
 
+//MARK: MonkeyChat
+extension ConversationsListViewController {
+
+  // Conversation
+  func getConversations(_ from:Double) {
+    
+    if self.isGettingConversations && !self.shouldRequestConversations {
+      return
+    }
+    
+    self.isGettingConversations = true
+    let conversations = DBManager.getConversations(self.conversationArray.last, count: 10)
+    
+    // Load from local database
+    if !conversations.isEmpty {
+      let delayTime = DispatchTime.now() + Double(Int64(0.5 * Double(NSEC_PER_SEC))) / Double(NSEC_PER_SEC)
+      DispatchQueue.main.asyncAfter(deadline: delayTime) {
+        for conversation in conversations {
+          //do not replace if the conversation already exists
+          if let conv = self.conversationHash[conversation.conversationId] {
+            conv.info = conversation.info
+            conv.members = conversation.members
+          }else{
+            self.conversationHash[conversation.conversationId] = conversation
+            self.conversationArray.append(conversation)
+          }
+        }
+        self.tableView?.reloadData()
+        self.isGettingConversations = false
+      }
+      return
+    }
+    
+    // Load from monkey store
+    Monkey.sharedInstance().getConversationsSince(from, quantity: 5, success: { (conversations) in
+      var idUsers = Set<String>()
+      for conversation in conversations {
+        //do not replace if the conversation already exists
+        if let conv = self.conversationHash[conversation.conversationId] {
+          conv.info = conversation.info
+          conv.members = conversation.members
+        }else{
+          self.conversationHash[conversation.conversationId] = conversation
+          self.conversationArray.append(conversation)
+        }
+        
+        if let msg = conversation.lastMessage {
+          DBManager.store(msg)
+        }
+        
+        idUsers.formUnion(Set(conversation.members as NSArray as! [String]))
+        DBManager.store(conversation)
+      }
+      
+      let unknownUsers = DBManager.monkeyIdsNotStored(idUsers)
+      if !unknownUsers.isEmpty {
+        Monkey.sharedInstance().getInfoByIds(unknownUsers, success: { (users) in
+          DBManager.store(users)
+          }, failure: { (task, error) in
+            print(error)
+        })
+      }
+      
+      if !conversations.isEmpty {
+        self.tableView?.reloadData()
+      }
+      
+      self.isGettingConversations = false
+      
+      self.refreshControl?.endRefreshing()
+      }, failure: { (task, error) in
+        self.isGettingConversations = false
+        self.refreshControl?.endRefreshing()
+        print(error)
+    })
+  }
+  
+  func createConversation(_ conversationId:String ,message:MOKMessage) {
+    let conversation = MOKConversation(id: conversationId)
+    var user = DBManager.getUser(conversationId)
```

Comment:
```swift
//maybe name it storedUser instead of user
guard let user = DBManager.getUser() else {
  //user doesn't exist
  //the only thing is that you won't be able to reuse the var named `user`

  
  return
}
```